package.json

```diff
@@ -118,5 +118,8 @@
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0"
+  },
+  "dependencies": {
+    "strip-ansi": "^6.0.0"
```

Comment:
is this dependency necessary for the whole monorepo? we should scope this down to the package that uses this

Reply:
I moved the dependency to terra-functional-testing

Reply:
https://github.com/cerner/terra-toolkit/pull/662/files#diff-72c2847b50d4152bd910ef25c33dafbd31a04d507329a333036e6226e5e8ef71R77